src/Controller/HomeController.php

```diff
@@ -2,17 +2,16 @@
 
 namespace TrendingRepos\Controller;
 
-use TrendingRepos\App;
 use TrendingRepos\Core\Paginator;
 use TrendingRepos\Core\Trending;
 
-class HomeController 
+class HomeController extends Controller 
 {
     public function home()
     {
         $content = Trending::fetch($this->getArgs());
-        $offset = $_GET['offset'] ?? App::get('config')['offset'];
-        $paginator = new Paginator;
+        $offset = $_GET['offset'] ?? $this->config['offset'];
```

Comment:
By the way, there's no such config value in `config.Example.php`. Can you update it to current state so I can also add new values to my local config?